cvat/apps/quality_control/quality_reports.py

```diff
@@ -1028,7 +1048,11 @@ def _match_segments(
                 extra_args["label_matcher"] = label_matcher
 
             returned_values = _match_segments(
-                a_objs, b_objs, distance=distance, dist_thresh=self.iou_threshold, **extra_args
+                a_objs,
+                b_objs,
+                distance=distance,
+                dist_thresh=dist_thresh or self.iou_threshold,
```

Comment:
```suggestion
                dist_thresh=dist_thresh if dist_thresh is not None else self.iou_threshold,
```

The value of `0` passed would be ignored.

Reply:
Ah, true.